src/components/Market.js

```diff
@@ -0,0 +1,51 @@
+
+import React, { Component } from 'react'
+import { observer, inject } from 'mobx-react'
+import Item from './Item'
+
+
+// Can be designed as a functional component that uses hooks instead
+class Market extends Component {
+    constructor(){
+        super()
+        this.state = {
+            input : ''
+        }
+    }
+
+    addItem = () =>{
+        const str = this.state.input
+        const itemData = str.split(',')
+        let price = itemData[1]
+        let quantity = itemData[2]
```

Comment:
you could do - 
`const [price, quantity] = this.state.input.split(',')`
instead of lines 17-20